myfunctions.js

```diff
@@ -198,9 +199,12 @@ const checkEmail = () => {
   return valid;
 };
 
-inputEmail.addEventListener('input', checkEmail);
-contactForm.addEventListener('submit', (event) => {
-  if (!checkEmail) {
-    event.preventDefault();
+const formSubmit = (e) => {
+  if (!checkEmail()) {
+    e.preventDefault();
+    inputEmail.classList.add('error')
```

Comment:
- [ ] If a user submits the contact form with the email field containing uppercase characters an error message should show up. (for now it's validating and making input as red, but don't you think it's better to give error message as well like `email should be in lowercase` for recruiter to know why this form has not submitted).
- [ ] The error message should show up close to the "submit" button.